### ISSUE TITLE
3.0-RC1: Controller changes "Page" mixin on second run

### ISSUE THREAD

**Issue by wernerkrauss** [silverleague/silverstripe-ideannotator#104]
After the first run on a SS4 project I get annotations like this:

```
 * Class \Foo\Pages\HomePageController
 *
 * @property \Foo\Pages\HomePage dataRecord
 * @method \Foo\Pages\HomePage data()
 * @mixin \Foo\Pages\HomePage
 ``` 

Running ideannotator again changes the `@mixin` line like this:
```
* @mixin \Foo\Pages\HomePage dataRecord
```

If it's not a bug, why doesn't it produce it the first run?

**Comment by Firesphere** [comment]
Yep, not yet documented but known (to me) issue where it doesn't properly add the `dataRecord` bit.

This inconsistent behaviour has existed since the beginning. I'm hoping upgrading `reflection-docblock` will solve this, but no promises. For now... Controllers have to deal with what they get :/

**Comment by Firesphere** [comment]
This seems to be related to #77

**Comment by Firesphere** [comment]
Should be resolved in #107

**Comment by Firesphere** [comment]
Resolved in #107, closing this issue.
If the issue still remains, please open a new issue
